packages/babel/README.md

```diff
@@ -0,0 +1,368 @@
+[npm]: https://img.shields.io/npm/v/@rollup/plugin-babel
+[npm-url]: https://www.npmjs.com/package/@rollup/plugin-babel
+[size]: https://packagephobia.now.sh/badge?p=@rollup/plugin-babel
+[size-url]: https://packagephobia.now.sh/result?p=@rollup/plugin-babel
+
+[![npm][npm]][npm-url]
+[![size][size]][size-url]
+[![libera manifesto](https://img.shields.io/badge/libera-manifesto-lightgrey.svg)](https://liberamanifesto.com)
+
+# @rollup/plugin-babel
+
+🍣 Seamless integration between Rollup and Babel.
+
+## Why?
+
+If you're using Babel to transpile your ES6/7 code and Rollup to generate a standalone bundle, you have a couple of options:
+
+- run the code through Babel first, being careful to exclude the module transformer, or
+- run the code through Rollup first, and _then_ pass it to Babel.
+
+Both approaches have disadvantages – in the first case, on top of the additional configuration complexity, you may end up with Babel's helpers (like `classCallCheck`) repeated throughout your code (once for each module where the helpers are used). In the second case, transpiling is likely to be slower, because transpiling a large bundle is much more work for Babel than transpiling a set of small files.
+
+Either way, you have to worry about a place to put the intermediate files, and getting sourcemaps to behave becomes a royal pain.
+
+Using Rollup with `@rollup/plugin-babel` makes the process far easier.
+
+## Requirements
+
+This plugin requires an [LTS](https://github.com/nodejs/Release) Node version (v8.0.0+) and Rollup v1.20.0+.
+
+## Install
+
+```bash
+npm install @rollup/plugin-babel --save-dev
+```
+
+## Usage
+
+Create a `rollup.config.js` [configuration file](https://www.rollupjs.org/guide/en/#configuration-files) and import the plugin:
+
+```js
+import babel from '@rollup/plugin-babel';
+import pkg from './package.json';
+
+const config = {
+  input: 'src/index.js',
+  output: [
+    {
+      file: pkg.module,
```

Comment:
Let's make this example simpler and just hardcode `dir: 'output'` like the other READMEs.

Reply:
I'm not sure if people expect to use `dir` here - isn't it targeting multi-entry scenarios (so a more complex one)? I often find people wanting to have control over the single output filename.

Reply:
@Andarist the real point of this section is an easy, frictionless intro into using the plugin. more complex, nuanced examples are typically located further down in either options sections or in "meta" sections after options that explain various concepts in more detail. this is a core tenant we've been following.